### PR TITLE
:book: Introduce logos into the book

### PR DESCRIPTION
**Pull request by DirectXMan12**
This introduces logos into the book.  The assets are stored in Netlify large media storage, and are copies of the publicly available assets stored in a GCS bucket.

### CODE REVIEW

**Comment by DirectXMan12** [comment]
Will introduce a logo in the readme in a bit

File: docs/book/src/logos/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+The official location for the logos is in a [public GCS
+bucket][kb-logos-gcs].
+
+These logos are copies used in the book, resized to their appropriate
+sizes. They're tracked with Netlify's *Large Media Storage*, which is
+a git-lfs implementation. Please do not try and use GitHub's git-lfs
+implementation to store these.
+
+[kb-logos-gcs]: https://console.cloud.google.com/storage/browser/kubebuilder-logos
```
Comment (mengqiy) [inline docs/book/src/logos/README.md]: This requires users to sign in with a GCP account.
Maybe better to use https://storage.googleapis.com/kubebuilder-logos/.

**Comment by mengqiy** [comment]
One nit. LGTM otherwise.

**Pull request merged**